### ISSUE TITLE
Fix ARM11 clock rate

### ISSUE THREAD

**Issue by TuxSH** [citra-emu/citra#2445]
Kernel11 uses in places such as SleepThread, to convert a time in ns to a number of ticks (then add the current tick count to the result), code like:

```cpp
      v7 = __PAIR__(
             (0x44A2FA85 * (unsigned __int64)(unsigned int)ns_high >> 32) + 0x44A2FA85 * (ns_high >> 31),
             0x44A2FA85 * ns_high)
         + (0x44A2FA85 * (unsigned __int64)(unsigned int)ns_low >> 32)// ns -> tick
         + KTimerAndWDTManager::GetSystemTick(&TimerAndWDTManager);
```

This is a multiply-by-constant division:

1 tick -> 2^32/0x44A2FA85  ns

Hence:
1/(2^32/0x44A2FA85) ticks -> 1 s

Therefore the theoritical clockrate of the 3DS's MPCore ARM11 processor is 268,111,855.956 Hz.

**Comment by SijmenSchoon** [comment]
Is this issue still relevant, or has #336 already fixed it?

**Comment by wwylele** [comment]
The fraction number is not necessarily an accurate number. It could be resulted from optimization from an integer division, in a similar way like this: https://godbolt.org/g/4KKs75. In this example, by doing the same math calculation purely from assembly code `input * 1459166399 / (2 ^54)`, one would conclude that the original divisor is `12345677.999320`, instead of the actual integer `12345678`. In the same way, the original kernel source code probably wrote an integer there, and due to the optimization, what we can calculate from the assembly is not the integer.

**Comment by FearlessTobi** [comment]
This is fixed by #3119
